content/applications/hr/employees/offboarding.rst

```diff
@@ -0,0 +1,143 @@
+revealing a drop-down menu. Click :guilabel:`# Departure Employee` in the list, then click away from
+the drop-down menu to close it. Now the report shows all the employees who were archived for the
+:guilabel:`Last 365 Days`.
+
+To view this information in an easier format, click the :icon:`oi-pivot` :guilabel:`(Pivot)` icon in
+the upper-right corner, and the data is presented in a pivot table. The various employees, organized
+by department, populate the rows. The columns display the following totals: the monthly
+:guilabel:`Wage`, :guilabel:`Fuel Card`, total :guilabel:`Annual Employee Budget` (also referred to
+as annual salary), the number of :guilabel:`New Employees`, as well as the number of
+:guilabel:`Departure Employees` (employees who left).
+
+.. image:: offboarding/pivot-departures.png
+   :align: center
+   :alt: The Employees Analysis report, modified to show departed employees only.
+
+Use case: view employees who left in the third quarter
+======================================================
+
+To view only the employees who have left the company during July, August, and September of the
+current year, first open the :ref:`Employees offboarding report <employees/offboarding-report>` and
+only display the information in a :icon:`oi-pivot` :guilabel:`(Pivot)` table.
+
+Next, click the :guilabel:`Measures` :icon:`fa-caret-down` button in the upper-left corner,
+revealing a drop-down menu. Click :guilabel:`# New Employees`, :guilabel:`Annual Employee Budget`,
+:guilabel:`Fuel Card`, and :guilabel:`Wage` in the list, to deselect these metrics and hide them in
+the table. Click away from the drop-down menu to close it. Now the report shows all the employees
+who left the company for the :guilabel:`Last 365 Days`.
+
+To only view the data for the third quarter, click the :icon:`fa-caret-down` :guilabel:`(down
+arrow)` in the search bar, revealing multiple filter and grouping options. Click :guilabel:`Last 365
+Days` in the :icon:`fa-filter` :guilabel:`Filters` column, to turn off that filter. Then click
+:guilabel:`Date`, and click :guilabel:`Q3` from the resulting drop-down menu. Click off of the
+drop-down menu to close it.
+
+Next, click :icon:`fa-minus-square-o` :guilabel:`Total` at the top of the rows to collapse it. After
+doing so, only one number appears on the pivot table: the total number of employees who left the
+company in the third quarter. In this example, there are ten.
+
+To view the various departments where employees left, click :icon:`fa-plus-square` :guilabel:`Total`
+at the top of the rows, revealing a drop-down menu, and click :guilabel:`Department`. Click away
+from the drop-down to close it, and now the pivot table displays the total number of employees who
+left (:guilabel:`# Departure Employee`) during the third quarter, organized by department.
+
+In this example, it can be determined that out of the tn employees who left, the
```

Comment:
I would consider turning this last section into an example admonition block since it is more specific than the rest, which is instructions on how to build the report. Since this image and the "in this example" parts are talking about a specific report, it would fit better in an example, than in general instructions.

Reply:
OK, after looking at this doc, I decided that this section (the reporting part) should be in its own "Employee Retention Report" doc, and the offboarding only should be in this doc. So- I am moving this to a new doc. I know, crazy! But it seems like it's a better fit (I like to have the reports separate). I needed a little while to make sense of this doc!